tests/snmpsim/kti.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.1.0|4|KGS-1060 GigaBit Ethernet Switch
```

Comment:
It's also worth switching the yaml discover to use sysObjectId if it's useful. Relying on sysDescr matches should be the last resort.